tools/percolator/msgf2pin.xml

```diff
@@ -0,0 +1,60 @@
+    <description>to create Percolator input files</description>
+    <requirements>
+        <requirement type="package" version="2.10">percolator_converters</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:"/>
+    </stdio>
+    <command interpreter="python">msgf2pin.py 
+             enzyme=$enzyme matches=$matches aa-freq=$aafreq PTM=$ptm PNGaseF=$pngase isotope=$isotope 
+             $pinout $target $decoy
+    </command>
+    <inputs>
+        <param name="target" type="data" format="mzid,meta2pin" label="Target MSGF+ results or metafile pointing to them" />
+        <param name="decoy" type="data" format="mzid,meta2pin" label="Decoy MSGF+ results or metafile pointing to them" />
+        <param name="enzyme" type="select" label="Enzyme used">
+          <option value="trypsin">trypsin</option>
+          <option value="no_enzyme">no enzyme</option>
+          <option value="elastase">elastase</option>
+          <option value="pepsin">pepsin</option>
+          <option value="proteinasek">proteinase K</option>
+          <option value="thermolysin">thermolysin</option>
+          <option value="chymotrypsin">chymotrypsin</option>
+          <option value="lys-n">lys-N</option>
+          <option value="lys-c">lys-C</option>
+          <option value="arg-c">arg-C</option>
+          <option value="asp-n">asp-N</option>
+          <option value="glu-c">glu-C</option>
+        </param>
+        <param name="matches" type="integer" optional="true" label="Max number of PSMs for a spectrum" />
+        <param name="aafreq" type="boolean" label="Calculate amino acid frequency features"/>
+        <param name="ptm" type="boolean" label="Calculate feature for number of PTMs"/>
+        <param name="pngase" type="boolean" label="Calculate feature based on N-linked glycosylation resulting from PNGaseF"/>
+        <param name="isotope" type="boolean" label="Calculate mass differnce to closest isotope instead of avg. mass"/>
+    </inputs>
+    <outputs>
+        <data format="percin" name="pinout"/>
+    </outputs>
+    <tests>
+        <test>
+          <param name="target" value="target.mzid"/>
+          <param name="decoy" value="decoy.mzid"/>
+        </test>
+    </tests>
+    <help>
+The percolator converter msgf2pin converts mzIdentML to input for percolator.
+Target and decoy inputs are MzIdentML-files of MS-GF+ from
+separate target and decoy searches. Internal MS-GF+ target/decoy
+analysis should be turned off, and the addFeatures options turned on.
+Multiple MzIdentML-files can be merged by replacing target and decoy
+filepaths with meta files. Meta files are text files containing the
+the paths of mzid-files, one path per line. For successful results,
+the different runs should be generated under similar conditions.
+    </help>
+    <citations>
+Viktor Granholm, Sangtae Kim, José C.F. Navarro, Erik Sjölund, Richard D. Smith, and Lukas Käll
```

Comment:
This does not work. `<citation type="doi">` is the way to go if you have a doi, otherwise you can specify a bibtex entry.